func/syshealth.sh

```diff
@@ -53,6 +53,7 @@ function syshealth_update_web_config_format() {
     write_kv_config_file
     unset system
     unset known_keys
+    unset domain_redirect
```

Comment:
Is not needed here. We should add Redirect + Redirect in here to add it if it doesn't exists